corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -19,6 +19,7 @@
 
 from casexml.apps.case import const
 from casexml.apps.case.xml.parser import case_update_from_block
+from custom.covid.utils import get_ush_extension_cases_to_close
```

Comment:
@sravfeyn I think you may have forgotten to commit this function

Reply:
Ah, thanks. I have added it now.

Reply:
Can this be renamed to something like `custom.covid.casesync` rather than adding yet another `utils` module.

Reply:
Good idea, sure.